src/components/ChangePlan/Card/Ribbon.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const Ribbon = ({ children, position }) => {
+  return <div className={`dp-ribbon dp-ribbon-${position}`}>{children}</div>;
```

Comment:
I think that it would be nice accepting a default position, would it work?

```suggestion
const Ribbon = ({ children, position = "top-rigth" }) => {
  return <div className={`dp-ribbon dp-ribbon-${position}`}>{children}</div>;
```

Reply:
changed! thanks